accounts/abi/bind/base.go

```diff
@@ -151,15 +152,25 @@ func DeployContract(opts *TransactOpts, abi abi.ABI, bytecode []byte, backend Co
 	return c.address, tx, c, nil
 }
 
+// DeployContractRaw deploys a contract onto the Ethereum blockchain and binds the
+// deployment address with a Go wrapper.  It expects its parameters to be abi-encoded
+// bytes.
+func DeployContractRaw(opts *TransactOpts, bytecode []byte, backend ContractBackend, packedParams []byte) (common.Address, *types.Transaction, *BoundContract, error) {
```

Comment:
How do you feel about moving this also to the v2 library? It feels weird that almost all the other utilities are there but to deploy I need to do from `accounts/bind`.

Reply:
SGTM